src/cmd-supermin-run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+if [ ! -d tmp ]; then
+    echo 'Need tmp/ dir for supermin to work' >&2
+    exit 1
+fi
+
+workdir=$(pwd)
+export workdir
+
+case "${CACHE:-}" in
```

Comment:
This is an env var that gets set before running `cosa supermin-run`? Myabe let's make it a little more specific?

```suggestion
case "${RUNVM_CACHE:-}" in
```